custom_components/evohome_cc/__init__.py

```diff
@@ -46,6 +56,33 @@
 SCAN_INTERVAL_DEFAULT = timedelta(seconds=300)
 SCAN_INTERVAL_MINIMUM = timedelta(seconds=10)
 
+ATTR_SYSTEM_MODE = "mode"
+ATTR_DURATION_DAYS = "period"
+ATTR_DURATION_HOURS = "duration"
+
+ATTR_ZONE_TEMP = "setpoint"
+ATTR_DURATION_UNTIL = "duration"
+
+SVC_REFRESH_SYSTEM = "refresh_system"
+SVC_SET_SYSTEM_MODE = "set_system_mode"
+SVC_RESET_SYSTEM = "reset_system"
+SVC_SET_ZONE_OVERRIDE = "set_zone_override"
+SVC_RESET_ZONE_OVERRIDE = "clear_zone_override"
+
+from .const import EVO_MODE_AWAY, EVO_MODE_CUSTOM, EVO_MODE_ECO, EVO_MODE_DAY_OFF, EVO_MODE_RESET, EVO_MODE_AUTO, EVO_MODE_HEAT_OFF
```

Comment:
These `const`s should come from **evohome_cc.const** _or_ **evohome_rf.const**, but not both, see above.